pkg/artifactory/resource_artifactory_local_alpine_repository.go

```diff
@@ -4,37 +4,39 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-var alpineLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{
-	"primary_keypair_ref": {
-		Type:     schema.TypeString,
-		Optional: true,
-		Description: "Used to sign index files in Alpine Linux repositories. " +
-			"See: https://www.jfrog.com/confluence/display/JFROG/Alpine+Linux+Repositories#AlpineLinuxRepositories-SigningAlpineLinuxIndex",
-	},
-}, compressionFormats)
-
 func resourceArtifactoryLocalAlpineRepository() *schema.Resource {
+	const pkt = "alpine"
```

Comment:
You do this repeatedly where you declare a constant, and then use it once. All that does is add an extra line of code. 

In this cause you could use it twice, but you're not. Get rid of this constant - all of them - they don't add any value

Reply:
Improved.
variable packageType used to replace literal `alpine` from all occurrences